Checkers/Game.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+using System.Linq;
+
+
+namespace Checkers
+{
+    class Game
+    {
+        public Game()
+        {
+            
+        }
+
+        // method to start the game from main
+        public void startGame()
+        {
+
+            
+                Board myBoard = new Board();
+                string nextMove = "";
+                myBoard.DrawBoard();
+                bool gameOver = false;
+
+                while(!gameOver)
+                {
+                    Console.WriteLine("Do you want to 'move' or 'remove' a checker?");
+                    nextMove = Console.ReadLine().ToLower();
```

Comment:
If you enter nothing, I believe `Console.ReadLine()` returns `null`. So add a null check (i.e. `?.`) before `.ToLower()`.